SumoLogic.Logging.Log4Net.Tests/BufferedSumoLogicAppenderTest.cs

```diff
@@ -185,10 +205,11 @@ private void SetUpLogger(long messagesPerRequest, long maxFlushInterval, long fl
             this.bufferedSumoLogicAppender.FlushingAccuracy = flushingAccuracy;
             this.bufferedSumoLogicAppender.RetryInterval = retryInterval;
             this.bufferedSumoLogicAppender.Layout = new PatternLayout("%m%n");
+            this.bufferedSumoLogicAppender.UseConsoleLog = true;
             this.bufferedSumoLogicAppender.ActivateOptions();
 
             this.log4netLog = LogManager.GetLogger(typeof(BufferedSumoLogicAppenderTest));
-            this.log4netLogger = this.log4netLog.Logger as Logger;

```

Comment:
With a safe cast, if `log4netLog.Logger` is not actually a `Logger` this will fail with a `NullReferenceException` on the next line when trying to access the `Additivity` property. This would be misleading about what the problem is. By directly casting it we'll get an `InvalidCastException` so we'll know immediately it's not a `Logger`.